qemu/tests/cfg/virtio_scsi_mq.cfg

```diff
@@ -8,6 +8,9 @@
     login_timeout = 240
     start_vm = no
     image_size_extra_images = 512M
+    irq_regex = "virtio\d+-request"
```

Comment:
Does the irq name of a virtio-net-pci match this as well?

Reply:
No, the irq name of virtio-net-pci is different with virtio-scsi-pci.